weather.php

```diff
@@ -1,30 +1,33 @@
 <?php
-  /**
-   * This page talks to weatherapi.com.
-   */
 
-  // API URL.
-  // You can register for free to get your own api key (fb0... below). 
-  $url = "http://api.weatherapi.com/v1/current.json?key=fb0b4c5c22d04c22be2202032210112&aqi=no&q=";
+/**
+ * This page talks to weatherapi.com.
+ */
 
-  // Check and filter query parameters.
-  $query = filter_input(INPUT_GET, 'query', FILTER_DEFAULT);
+// API URL.
+// You can register for free to get your own api key (fb0... below). 
+$currentUrl = "http://api.weatherapi.com/v1/current.json?key=fb0b4c5c22d04c22be2202032210112&aqi=no&q=";
```

Comment:
Doesn't look like you use currentUrl. I didn't notice... are you just showing both (current and three day) for all requests? If so, the choice in the web page may not make sense. Let's just show the selected one.

Reply:
Correct, the forecastUrl handles both Current and Forecast weather. The toggle simply switches visibility between the appropriate data from the payload.